sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -132,6 +132,18 @@ export async function drainAllMessages(receiver: Receiver<{}>): Promise<void> {
   await receiver.close();
 }
 
+/**
+ * Returns a TestClientType for either a Queue or a Subscription
+ * @param useSessions 
+ */
+export  function getRandomeReceiverTestClientType(useSessions: boolean): TestClientType {
+  const noSessionRecieverClientTypes = [TestClientType.PartitionedQueue, TestClientType.PartitionedSubscription, TestClientType.UnpartitionedQueue, TestClientType.UnpartitionedSubscription];
+  const withSessionRecieverClientTypes = [TestClientType.PartitionedQueueWithSessions, TestClientType.PartitionedSubscriptionWithSessions, TestClientType.UnpartitionedQueueWithSessions, TestClientType.UnpartitionedSubscriptionWithSessions];
+
+  const index = Math.floor(Math.random() * noSessionRecieverClientTypes.length);
```

Comment:
Rather than being totally random can we base it on something that's a bit more stable? Otherwise it'll be difficult to chase CI failures if we get it only during a specific branch (unlikely, but it'd be nice to never have that issue).

Perhaps we could base it on the day or something of that nature?

Reply:
I made sure that the test title reflect the TestClientType. This is exactly for the case you mentioned i.e. to chase down CI test failures and identify the kind of queue/subscription being used just in case the issue does not repro with the randomly choses queue/subscription locally.

If we want to target a particular type when running locally, we can edit the test and use TestClientType enum directly

Reply:
It'd be nice to not have to push temp changes if we can avoid it. Is there a reason we're averse to having it pick the same set of tests on the same day?